### ISSUE TITLE
How do you continue your epoch dataload when iteration %epoch_size==0?

### ISSUE THREAD

**Issue by onbigion13** [amdegroot/ssd.pytorch#529]
In you code, I see that you do the running loop by the iterating of the variable "iteration" from "args.start_iter" to  "cfg['max_iter']".
but I see that the declaration of dataloader—— data_loader = data.DataLoader(dataset, args.batch_size,                num_workers=args.num_workers,   shuffle=True, collate_fn=detection_collate,     pin_memory=True).
there's nothing special of the dataset VOC0712's declaration. so how do you continue feeding new batches into the model when iteration % epoch_size==0??? it puzzled me few days. I cannot find where you handle the dataloading when itertation%epoch_size==0.

**Comment by FathiMahdi** [comment]
@onbigion13 have you understand this issue ?
